Resizable: add demo description (#1779)
DevExpress/devextreme-demos: JSDemos/Demos/Resizable/Overview/description.md

If you want to enable live resize operations for a UI element, wrap that element into a Resizable widget. In this demo, you can resize the DataGrid. Try to drag the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) on the grid's edges.

Configure the following properties to specify resize operation constraints:

- [minWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minWidth)    

- [maxWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxWidth)    

- [minHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minHeight)    

- [maxHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxHeight)    

- [area](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#area)    

> You can display resize handles on edges or corners. Use the following keywords to set up the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property: *top*, *bottom*, *left* and *right*. If you specify two adjacent sides (for example, "bottom right"), the control displays a handle in the corner.

Reviewer: ```suggestion
You can display resize handles on edges or corners. Use the following keywords to set up the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property: *top*, *bottom*, *left*, and *right*. If you specify two adjacent sides (for example, "bottom right"), the control displays a handle in the corner.
```

The Resizable enables you to resize its content. In this demo you can drag [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) to resize the DataGrid.

> Configure four properties to specify the boundaries to which you can drag the handles:

Reviewer 2: ```suggestion
Configure the following properties to specify the boundaries to which you can drag the handles:
```

> The Resizable enables you to resize its content. In this demo you can drag [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) to resize the DataGrid.

Reviewer 3: (Articles) I can see that the previous version also referenced this component/widget as "The Resizable", but that doesn't match our usual practices. Either you say something like this "The Resizable widget" or you simply say "Resizable" without articles. 

(TextFlow) Your version starts with the Resizable widget and then refers to "its content". That's not immediately clear. (Can I specify it's own content - such as text and images maybe?) And then the second sentence doesn't flow well from the first. I feel that the we better explain this widget if we talk about other elements right away. The developer's goal is to enable resize operation for some other element. If they want it, they use Resizable. So the train of thought goes like this: "you want to let users resize an element" -> "wrap into resizable" -> "this demos shows a data grid example (i.e. locate the grid in the demo)" -> "you can drag handles to try and resize". 

```suggestion
If you want to enable live resize operations for a UI element, wrap that elements into a Resizable widget. In this demo, you can resize the DataGrid. Try to drag the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) on the grid's edges.
```

> Configure the following properties to specify the boundaries to which you can drag the handles:

Reviewer 3: (Concise Text) The word "which" is usually a flag that tells you there might be a way to rewrite the text shorter (or in a more straightforward manner). Here's my suggestion:

```suggestion
Configure the following properties to specify resize operation constraints:
```

- [minWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minWidth)    

> The lower width boundary.

Reviewer 3: I'm not sure you even need these descriptions. The property names are self explanatory. And the expressions like "lower width boundary" is just a way to say "minimum width" while trying to avoid repetition. But the phrase doesn't sound right to me. I would simply leave the list of properties without any additional comments.

- [maxWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxWidth)    
The upper width boundary.

- [minHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minHeight)    
The lower height boundary.

- [maxHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxHeight)    
The upper height boundary.

> You can specify four types of handles: *top*, *bottom*, *left* and *right*. Separate handles by a space in the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property to specify which handles to display in the UI. Combine nearest handles (for example, "bottom left") to show corner handles.

Reviewer 3: A few notes: 

- You say "four types". But you also say that if I combine values ("bottom right"), I can display corner handles. To that amounts to eight types of handles total (four on edges and four in corners). Obviously you can't display them all at the same time, but those are the possible types, it seems. 
- I wouldn't explain the separator character here in the demo description, for two reasons. One - they can look it up if they need. Two - you give an example that uses a space, so that already gives readers a hint. 
- "bottom left" is not the best example. The most commonly used handle is "bottom right". I can see it right now in this GitHub editor. So I would use that as an example. 

All in all, these comments made me come up with the following suggestion:

```suggestion
You can display resize handles on edges or corners. Use the following keywords to set up the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property: *top*, *bottom*, *left* and *right*. If you specify two adjacent sides (for example, "bottom right"), the control displays a single handle in the corner.
```

> You can specify four types of handles: *top*, *bottom*, *left* and *right*. Separate handles by a space in the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property to specify the handles that you want to show to users. Combine nearest handles (for example, "bottom left") to get corner handles.

Reviewer 2: ```suggestion
You can specify four types of handles: *top*, *bottom*, *left* and *right*. Separate handles by a space in the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property to specify which handles to display in the UI. Combine nearest handles (for example, "bottom left") to show corner handles.
```

> The [keepAspectRatio](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#keepAspectRatio) property specifies whether to resize the Resizable's content proportionally with corner handles. If you need to disable propotional resize, set this property to **false**.

Reviewer 3: (Concise Text) - in the first sentence I want to rephrase the last part: "resize the Resizable's content proportionally with corner handles". 
(Repetition) - then I can avoid the repetition of the word "proportional" in the second sentence. ("Free-form" - is one of the picture resize modes in iOS.)

```suggestion
The [keepAspectRatio](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#keepAspectRatio) property specifies whether a corner handle resizes content proportionally. If you need to allow free-form resize operations, set this property to **false**.
```

> Use the [area](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#area) property to keep the component's resize in a specific area.

Reviewer 2: ```suggestion
Use the [area](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#area) property to enable users to resize the component within a specific area.
```

> Use the [area](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#area) property to enable users to resize the component within a specific area.

Reviewer 3: I don't know exactly what this property does... I tried to look it up in the following topic, but I guess it's a new member: 

https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxResizable/

I assume it may be somehow related to minWidth, minHeight, maxWidth, and maxHeight. In this paragraph should be next to the one that describes those size constraint properties.
